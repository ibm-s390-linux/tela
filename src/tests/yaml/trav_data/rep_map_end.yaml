map1:
  child1:
replace:
  child2:
