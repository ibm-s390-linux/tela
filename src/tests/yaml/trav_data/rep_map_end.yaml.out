Before (root=non-null):
==================================
  map1:
    child1:
  replace:
    child2:
==================================

Callback:
==================================
map1                                    : map1:
map1/child1                             :   child1:
replace                                 : replace:
*** Replacing node
==================================

After (root=non-null):
==================================
  map1:
    child1:
  replacement
==================================
